GSA/digitalgov.gov
# Update COP Card Mobile Layout

## Summary

This Pr will address the bug on the homepage for Community of Practice by turning it into a one column layout while in mobile view.

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-cop-mobile-fix/)



### Solution

Since the group of cards was set to a column format once it hit a mobile break point it was not able to properly wrap it to fit the screen.  This will remove that setting and have it set to just one column to fit on the screen. 

Before: 
![Screenshot 2024-02-12 at 2 42 55 PM](https://github.com/GSA/digitalgov.gov/assets/88721460/cef8c9fa-a373-4a05-b9d1-c40a52ad9bdd)


After:
 
![Screenshot 2024-02-12 at 2 22 02 PM](https://github.com/GSA/digitalgov.gov/assets/88721460/4ba5bd28-68f7-438d-b75a-e409bd19de58)


### How To Test

1. Visit [preview link](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-cop-mobile-fix/) and inspect your browser and reduce the screen size to a min of 320 pixels and check that there are no wrapping issues with the COP card group on the home page. 
2. Visit the preview link on a mobile device and verify the same as above. 

---

### Dev Checklist
